rmgpy/data/kinetics/family.py

```diff
@@ -1754,11 +1757,13 @@ def _create_reaction(self, reactants, products, is_forward):
 
         # Store the labeled atoms so we can recover them later
         # (e.g. for generating reaction pairs and templates)
-        labeled_atoms = []
+        reaction.labeled_reactant_atoms, reaction.labeled_product_atoms = list(), list()
         for reactant in reaction.reactants:
             for label, atom in reactant.get_all_labeled_atoms().items():
-                labeled_atoms.append((label, atom))
-        reaction.labeledAtoms = labeled_atoms
+                reaction.labeled_reactant_atoms.append((label, atom))
```

Comment:
Would this not be easier to work with as a dictionary than a list of tuples?

Reply:
Absolutely agree. I'll make the change.